src/apis/teams/types.ts

```diff
@@ -0,0 +1,31 @@
+export type GetTeamResponse = {
+  team_members_meta: GetTeamMemberResponse[];
+  created_date: string;
+  modified_date: string;
+  pk: string;
+  title: string;
+  about: string;
+  responsibilities: string[];
+  github: string;
+  discord: string;
+};
+
+export type GetTeamMemberResponse = {
+  user: string; // foreign key to user
+  is_lead: boolean;
+  team: string; // foreign key to team
+  pk: string;
+  job_title: string;
+  modified_date: string;
+  created_date: string;
+};
+
+export type GetCoreTeamResponse = Omit<GetTeamResponse, 'team_members_meta'> & {
+  core_members_meta: GetCoreTeamMemberResponse[];
+};
```

Comment:
I didn't know about this. Niceeeee!

Reply:
noiceee, we all can learn from one another 👍🏻